.github/workflows/build.yml

```
@@ -328,3 +332,51 @@ jobs:
       - name: Upload release
         run: |
           aws s3 sync --acl public-read dist s3://static.realm.io/ci/test/${{ env.PACKAGE_VERSION }}/
+      - name: Publish Github Release
+        uses: ncipollo/release-action@3ac4132803a6419fa2a7f4e9dbd1d93fceb690b9
+        with:
+          artifacts: "dist/*.exe,dist/*.dmg,dist/*.zip,dist/*.AppImage"
+          bodyFile: dist/RELEASENOTES.md
+          name: ${{ steps.get-version.outputs.version }}
+          commit: main
+          tag: ${{ steps.get-version.outputs.version }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          draft: false
+      - name: Update Changelog
+        run: |
+          echo "## vNext (TBD)
+
+          ### Enhancements
+          * None
+
+          ### Fixed
+          * None
+
+          ### Internals
+          * Updated dependencies
+          " | cat - CHANGELOG.md >> temp
+          mv temp CHANGELOG.md
+        shell: bash
+      - name: Create vNext PR
+        id: vnext-pr
+        uses: peter-evans/create-pull-request@6c704eb7a8ba1daa13da0dcea9bb93a4fe530275
+        with:
+          branch: prepare-vnext
+          title: Prepare for vNext
+          body: Update Changelog for vNext
+          delete-branch: true
+          base: main
+          commit-message: Prepare for vNext
+      - name: Merge Pull Request
+        uses: juliangruber/merge-pull-request-action@333730196b34b74936aad75a4e31c23a57582d14
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          number: ${{ steps.vnext-pr.outputs.pull-request-number }}
+          method: squash
+      - name: 'Post to #realm-releases'
```

This is not the channel that it'll post in anyway.

```suggestion
      - name: 'Post to Slack'
```